app/Http/Controllers/UsersController.php

```diff
@@ -97,20 +97,20 @@ public function store(UserRequest $request)
      */
     public function show($id)
     {
+        $activities = [];
+
         $user = $this->repository->find($id);
 
-        $competitions = User::find($id)->competitions;
+        $competitions = User::findOrFail($id)->competitions;
 
-        // Get the user's reportback for each competition they are in.
         foreach ($competitions as $competition) {
-            $reportback = $this->manager->getActivityForUser($id, $competition);
+            $parameters = $this->manager->getCampaignParameters($competition);
+            $parameters['users'] = $user->id;
 
-            if ($reportback) {
-                $competition->reportback = $reportback;
-            }
+            $activities[] = $this->manager->appendReportback($competition, $parameters);
```

Comment:
Ended up saving the updated competition to the activities array, but could potentially just be reduced to override the competition in the collection? Thoughts?